packages/grafana-schema/src/veneer/common.types.ts

```diff
@@ -0,0 +1,8 @@
+import * as raw from '../common/common.gen';
```

Comment:
Not important for this pr, but long term we should try to avoid the whole `import * as x from 'package'` pattern since it makes the tree shaking a bit trickier.

Reply:
I was wondering since it's only types do we even need tree shaking? I realise that in this case we also have default values there but if those were separated would that eliminate the need for tree shaking?